### PR TITLE
add `gantt chart` specs

### PR DESCRIPTION
**Pull request by theiliad**
- Updates sidenav items
    - removes `complex charts`
    - adds
        - Flow & Relationship Charts
        - Hierarchical & Spatial Charts
        - Gantt charts

### CODE REVIEW

**Review by guidari — APPROVED** [review]
LGTM!

File: src/pages/data-visualization/gantt-charts/index.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+[carbon-charts repository](https://github.com/carbon-design-system/carbon-charts).
+
+</InlineNotification>
+
+## Resources
+
+<Row className="resource-card-group">
+  <Column colLg={4} colMd={4} noGutterSm>
+    <ResourceCard
+      subTitle="Figma kit"
```
Comment (aagonzales) [inline src/pages/data-visualization/gantt-charts/index.mdx:28]: Typically, we put the title of the kit in the card name. And the Figma icon denotes its a figma file

```suggestion
      subTitle=Gantt Chart Kit (Alpha)"
```

**Review by aagonzales — CHANGES_REQUESTED** [review]
Looks good just one small thing

File: src/pages/data-visualization/gantt-charts/index.mdx (hunk 1 above)
Comment (theiliad) [reply src/pages/data-visualization/gantt-charts/index.mdx]: thanks, addressed!

**Review by aagonzales — APPROVED** [review]

**Pull request merged**